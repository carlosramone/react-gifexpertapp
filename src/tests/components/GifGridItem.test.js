import React from "react";
import { shallow } from "enzyme";
import { GifGridItem } from "../../components/GifGridItem";

describe("Pruebas GifGridItem", () => {
	const wraper = shallow(<GifGridItem title="TÃ­tulo" url="https://localhost/imagen.jpg" />);

	// beforeEach(() => {
	// 	wraper = shallow(<GifGridItem />);
	// });

	test("Debe mostrar <GifGridItem/> correctamente", () => {
		expect(wraper).toMatchSnapshot();
	});
});
