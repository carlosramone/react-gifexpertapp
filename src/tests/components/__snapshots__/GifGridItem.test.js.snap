// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas GifGridItem Debe mostrar <GifGridItem/> correctamente 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Título"
    src="https://localhost/imagen.jpg"
  />
  <p>
    Título
  </p>
</div>
`;
